boards/arm64/rpi_5/doc/index.rst

```diff
@@ -159,3 +159,35 @@ When you power on the Raspberry Pi 5, you will see the following output in the s
 
 .. _Raspberry Pi Debug Probe:
    https://www.raspberrypi.com/products/debug-probe/
+
+XEN Dom0
```

Comment:
Also, I think it is better to make these changes as a part of corresponding commits, which added these functionality (it will be easier to re-arrange it if we will make e.g. DomD snippet as a separate project).

Reply:
I'll keep it in mind.